Change op_time from range slider to input boxes

linneahallin opened the pull request.

linneahallin committed:
Change op_time from range to input boxes

raoulkent commented on components/op_time.py:
```diff
@@ -19,12 +20,52 @@ def get_component():
                 ),
                 html.Hr(style={"margin-top": 0, "margin-bottom": 10}),
-                OpTime._op_time_slider(),
+                # OpTime._op_time_slider(),
```
```suggestion
                
```
Remove?

raoulkent commented on components/op_time.py:
```diff
@@ -42,28 +83,84 @@ def _op_time_slider():
         )
         return widget
 
+    # @staticmethod
```
Remove commented out code?

raoulkent reviewed (changes requested):
Will check detailed (local sys) soon

raoulkent committed:
Refactor

Increase readability.
Remove unused code.
Fix near-infinite upper bound for time limit.

raoulkent reviewed (approved):
I've made a minor refactor, just check that before merging.
The only functional change is that of using `sys.maxsize` as the upper bound for range when filtering for time.
